cmd/crc/cmd/status.go

```diff
@@ -89,19 +86,29 @@ func (s *status) prettyPrintTo(writer io.Writer) error {
 	}
 	w := tabwriter.NewWriter(writer, 0, 0, 1, ' ', 0)
 
-	lines := []struct {
+	// todo: replace this with Pair when switching to go@1.18
+	type line struct {
 		left, right string
-	}{
+	}
+	lines := []line{
 		{"CRC VM", s.CrcStatus},
-		{"OpenShift", openshiftStatus(s)},
-		{"Podman", s.PodmanVersion},
-		{"Disk Usage", fmt.Sprintf(
+	}
+
+	if s.OpenShiftVersion != "" {
+		lines = append(lines, line{"OpenShift", openshiftStatus(s)})
+	}
+	if s.PodmanVersion != "" {
+		lines = append(lines, line{"Podman", s.PodmanVersion})
+	}
+
```

Comment:
Fwiw, I would keep the change modifying `crc status` output separate from the change making use of the daemon for the status command. It's easier to revert one without reverting the other if one of these changes proved problematic. 
One thing which could be added is the OKD version (can be a follow-up if you want to focus on the current PR without adding more changes), it's currently missing.